examples/Makefile

```diff
@@ -146,6 +146,27 @@ get-ruby-custom-port:
 get-ruby-custom-port-verify:
 	kubeless function call get-ruby-custom-port |egrep hello.world
 
+get-php:
+	kubeless function deploy get-php --trigger-http --runtime php7.2 --handler helloget.foo --from-file php/helloget.php
+	echo "curl localhost:8080/api/v1/proxy/namespaces/default/services/get-php/"
+
+get-php-verify:
+	kubeless function call get-php | egrep "hello world"
+
+get-php-deps:
+	kubeless function deploy get-php-deps --trigger-http --runtime php7.2 --handler hellowithdeps.foo --from-file php/hellowithdeps.php --dependencies php/composer.json
+	echo "curl localhost:8080/api/v1/proxy/namespaces/default/services/get-php-deps/"
+
+get-php-deps-verify:
+	kubectl logs -l function=get-php-deps | egrep "Hello"
+
+post-php:
+	kubeless function deploy post-php --trigger-http --runtime php7.2 --handler hellowithdata.foo --from-file php/hellowithdata.php
+	echo "curl --data '{\"hello\":\"world\"}' localhost:8080/api/v1/proxy/namespaces/default/services/post-php/ --header \"Content-Type:application/json\""
+
+post-php-verify:
+	kubeless function call post-php --data '{"it-s": "alive"}'| egrep "it.*alive"
+
```

Comment:
we need one last test to check that the timeout is working as expected. Something like:

```
timeout-php:
	$(eval TMPDIR := $(shell mktemp -d))
	printf '<?php\n function foo() { while(1) {} } }\n' > $(TMPDIR)/hello-loop.php
	kubeless function deploy timeout-php --trigger-http --runtime php7.2 --handler helloget.foo  --from-file $(TMPDIR)/hello-loop.php --timeout 4
	rm -rf $(TMPDIR)

timeout-php-verify:
	$(eval MSG := $(shell kubeless function call timeout-php 2>&1 || true))
	echo $(MSG) | egrep Request.timeout.exceeded
```

(you need to add this test to the `integration-tests.bats` file as well)

Reply:
Thanks, added!